### PR TITLE
ImageRef from latest complete release

### PR DESCRIPTION
**Pull request by rugwirobaker**

### CODE REVIEW

File: api/resource_releases.go [hunk 1]
```diff
@@ -40,35 +40,33 @@ func (c *Client) GetAppReleasesNomad(ctx context.Context, appName string, limit
...
-				releases: releasesUnprocessed(first: $limit) {
-					nodes {
-						id
-						version
-						description
-						reason
-						status
-						imageRef
-						stable
-						user {
```
Comment (dangra) [inline api/resource_releases.go:56]: removing user here will break `fly apps releases` output

Comment (rugwirobaker) [reply api/resource_releases.go:56]: Haaa, never meant to make that change, I'm pushing a fix

File: internal/command/scale/count_machines.go [hunk 2]
```diff
@@ -19,19 +20,27 @@ import (
...
 		return err
 	}
 
-	if len(machines) == 0 {
-		// We need at least one machine to grab the image to use.
-		// FIXME: fetch image, release id and version from latest "complete" release
-		return fmt.Errorf("there are no active machines for this app. Run `fly deploy` to create one and rerun this command")
+	var latestCompleteRelease api.Release
+
+	releases, err := apiClient.GetAppReleasesMachines(ctx, appName, "complete", 10)
```
Comment (dangra) [inline internal/command/scale/count_machines.go:35]: why 10 and not just 1 ? the latest?

File: internal/command/scale/count_machines.go [hunk 3]
```diff
@@ -19,19 +20,27 @@ import (
 )
 
 func runMachinesScaleCount(ctx context.Context, appName string, appConfig *appconfig.Config, expectedGroupCounts map[string]int, maxPerRegion int) error {
-	io := iostreams.FromContext(ctx)
+	var io = iostreams.FromContext(ctx)
```
Comment (kzys) [inline internal/command/scale/count_machines.go:23]: You don't need this change.

**Review by kzys — COMMENTED** [review]
For the lint errors, you should be able to do `gofmt -w $(find . -name '*.go')`

**Comment by dangra** [comment]
or Install [precommit](https://pre-commit.com/) hooks and let it yell and format the files for you.

**Review by dangra — APPROVED** [review]

**Pull request merged**